### PR TITLE
fix the error of using invalid key 'force' in nvim_set_hl function

### PR DESCRIPTION
**Pull request by Aliqyan-21**
## Checklist 
- [x] I read the [CONTRIBUTING GUIDE](https://github.com/slugbyte/lackluster.nvim/blob/main/CONTRIBUTING.md)
- [x] I opened a CONTRIBUTING ISSUE before opening the PR

## Description
I found the error for which I have raised the issue.

### CODE REVIEW

File: lua/lackluster/init.lua [hunk 1]
```diff
@@ -73,134 +73,134 @@ local USER_CONFIG = nil
...
+		if highlight_group.dont_skip or is_plugin_enabled then
+			for _, hl_spec in ipairs(highlight_spec_list) do
+				local hl_name = hl_spec.name
+				if dedup_set[hl_name] then
+					vim.notify("error: duplicate hi_spec :: " .. hl_name, vim.log.levels.ERROR)
+				else
+					dedup_set[hl_name] = true
+					hl_spec.name = nil -- must set to nil so that nvim_set_hl doesn't freq out
+					---@diagnostic disable-next-line: param-type-mismatch
+					vim.api.nvim_set_hl(0, hl_name, hl_spec)
```
Comment (slugbyte) [inline lua/lackluster/init.lua:199]: In your bug report #44 it looks like the affending error is on line 199 but I dont see any changes that would affect this call to `nvim_set_hl`

what exactly did you change that is supposed to fix this issue?

**Review by slugbyte — CHANGES_REQUESTED** [review]
I cannot see any changes to this file that have any impact

Sorry, I cant tell how this PR issue fixes your #43

The only things that I can tell have changed in your PR are that  white space is now 2 spaces tabs instead of 4, and double quotes have been changed to single quotes.

Am I missing something? I cant tell what change you have made to fix an issue. 
Can you explain what you changed and how you think it fixed your bug?  

Thank you so much for the bug report, and the patch! I hope we can resolve this together :)

**Comment by Aliqyan-21** [comment]
Hey slugbyte sir, 
sorry for my mistake earlier but now if you see in my commit [ca9ce05](https://github.com/slugbyte/lackluster.nvim/pull/44/commits/ca9ce059f34122a8997daa3875c25395c047dd56) 
you will see that I have fixed the issue properly. And now the error does not show in my neovim and the theme works.

**Comment by slugbyte** [comment]
I just tested on nvim versions `0.9.1` `0.9.2` `0.9.4` `0.9.5 0`  and `0.10.0` and found out that `force` only works on version 10 and above

I also realized I only have one highlight that uses `force` and it is unnecessary so I removed it :)

**Review by slugbyte — APPROVED** [review]

**Pull request merged**